lib/logging.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const util = require('util');
+const debug = require('debug');
+
+const prefix = 'metrics';
+
+/**
+ * A prefixed instance of the `debug` logger. You can call this directly, or
+ * call `extend()` on it to create a nested logger.
+ * @type {debug.Debugger}
+ */
+const logDebug = debug(prefix);
```

Comment:
Forgot to note that I started doing JSDoc-style comments in this new module in anticipation of #86, since the TypeScript compiler can build type definitions from JSDoc comments but not from the style of comments currently used in other files.